cymetric/cyclus.pyx

```diff
@@ -46,6 +47,24 @@ cdef class _Datum:
     #        free(self.ptx)
 
     def add_val(self, const char* field, value, shape=None, dbtype=cpp_typesystem.BLOB):
+        """Adds Datum values for type conversion.
```

Comment:
The 'for type conversion' here doesn't make a whole lot of sense.  Though this does do type conversion, what is important is that it adds a value to the current record.